src/main/java/org/fundacionjala/at15/katas/pokerhands/mauricio/Card.java

```diff
@@ -0,0 +1,77 @@
+package org.fundacionjala.at15.katas.pokerhands.mauricio;
+
+public  class Card {
+    private String value;
+    private String suit;
+    private final int cardValue1 = 1;
```

Comment:
names of the constants improved, good! You can also consider define them as static so they are accesible outside from this class in case you need them